certbot/storage.py

```diff
@@ -256,8 +259,26 @@ def __init__(self, config_filename, cli_config):
         self.live_dir = os.path.dirname(self.cert)
 
         self._fix_symlinks()
+        if update_symlinks:
+            self._update_symlinks()
         self._check_symlinks()
 
+    def __str__(self):
+        """Returns a string representation of the object."""
+        # import ipdb; ipdb.set_trace()
+        outdict = {"Lineage": self.lineagename,
+                   "Domains": " ".join(self.names())}
+        return "RenewableCert(%s)" % str(outdict)
```

Comment:
I think it would be very valuable to have a machine-readable output and I think people would use it a lot. It could be tab-delimited and maybe include

`lineage	expiry	domain:domain:domain:domain`

or maybe also some representation of the issuer and subject keys?

There could be separate human-readable and machine-readable versions of the output.

For the version that you show above, I would get rid of the quotation marks and also provide a human readable summary of the time to expiry, maybe like

    Valid until: Jan 24 21:28:00 2017 GMT  (89 days from now)

It would also be helpful to have a version for certificates that have already expired, maybe like:

    Valid until: Jan 24 21:28:00 2016 GMT  (EXPIRED)